packages/plugins/plugin-table/src/components/TableContainer.tsx

```diff
@@ -39,7 +39,14 @@ const TableContainer = ({ role, table }: LayoutContainerProps<{ table: TableType
   const queriedObjects = useQuery(space, schema ? Filter.schema(schema) : () => false, undefined, [schema]);
   const filteredObjects = useGlobalFilteredObjects(queriedObjects);
 
-  const handleDeleteRow = useCallback((row: any) => space?.db.remove(row), [space]);
+  // TODO(burdon): Undo action.
```

Comment:
use space plugin intent instead of directly removing, it has undo

Reply:
@ZaymonFC